apps/src/blockly/addons/blockSvgLimit.ts

```diff
@@ -0,0 +1,144 @@
+import {BlockSvg} from 'blockly';
+
+const BUBBLE_SIZE = 18;
```

Comment:
If you wanted to, I think you could add these to the constructor and make them
`private readonly bubbleSize: number = 18;`

Reply:
Sounds good!